src/buildkite_test_collector/pytest_plugin/failure_reasons.py

```diff
@@ -0,0 +1,113 @@
+"""Buildkite Test Engine PyTest failure reason mapping"""
+
+from __future__ import annotations
```

Comment:
This is just for Python 3.9 which we'll drop pretty soon, it's EOL in a couple of months.